core/src/main/scala/com/typesafe/tools/mima/core/MemberInfo.scala

```diff
@@ -7,8 +7,9 @@ object MemberInfo {
  /** The index of the string $_setter_$ in this string */
   private def setterIdx(name: String) = name.indexOf(setterTag)
 
-  private val setterTag = "$_setter_$"
+  private val setterTag = "$_setter_"
```

Comment:
oh, there's already an issue on this, it's #50 (@retronym helped me find it). I'll comment there.

in any case, it's actually irrelevant to this PR now, Bjorn removed the change in question in the latest version.